sdf/deployment.mdx

```diff
@@ -1,75 +1,68 @@
 ---
 title: Deployment
-description: Deployment/Worker Management
+description: Deployment of dataflow thru Worker
 sidebar_position: 60
 ---
 
-# Deployment/Worker Management
+# Introduction
 
-There are two modes to run dataflow in SDF: ephemeral and worker.  When [CLI] is run in ephemeral mode, it will run the dataflow in the same process as the CLI.  The dataflow will be terminated upon exiting CLI.  This is useful for development dataflow as well as testing the package.  
+When you use `run` command to execute a dataflow, it runs within the same process as the CLI. This is useful for development and testing because it's easy to start without needing to manage additional resources. It allows for quick testing and validation of the dataflow, and you can easily load and integrate development packages.
 
-Worker is a process that runs the dataflows continuously.  Unlike ephemeral CLI, worker is long running process.  That means, when you exit CLI, dataflow does not terminate.  It is primary used for running dataflow in production environment.  It can also be used in development environment to run dataflow when there is no need to test package.
+For production deployment, the `deploy` command is used to deploy the dataflow on a worker. All operations available in `run` also apply to deploy, with the following differences:
+- The dataflow is executed on the worker, not within the CLI process. The CLI communicates with the worker on the user's behalf.
+- The dataflow continues running even if the CLI is shut down. It will only terminate if the worker is stopped, shut down, or the dataflow is explicitly stopped or deleted.
+- Dataflows in the worker only have access to published packages, unlike `run` mode, which allows access to local packages. If you need to use a package, you must publish it first.
```

Comment:
This isn't true at the moment, is it going to be?

Reply:
this is how dataflow behave in the worker now.  did you see difference?

Reply:
I see. I misread "packages" as "operators"